packages/custom-field/theme/material/vaadin-custom-field-styles.js

```diff
@@ -24,6 +24,11 @@ registerStyles(
       -moz-osx-font-smoothing: grayscale;
     }
 
+    /* TODO: remove when all components use new indicator */
+    [part='label']::after {
```

Comment:
I'm not sure if I understand why `::after` needs to be manually hidden here.

Do the old components import the new styles?

Reply:
The original CSS for both old and new indicators comes from `@vaadin/vaadin-lumo-styles/mixins/required-field.js`

So we need to override it here for the time being, as I don't want to touch old implementations at this point.
Eventually we will update all the components to use new markup, and drop old code from the Lumo package.

Reply:
Understood. Thanks!